### ISSUE TITLE
loss optimizer clarification

### ISSUE THREAD

**Issue by abaqusstudent** [KevinMusgrave/pytorch-metric-learning#468]
Hi there

I see in some tutorials, the loss optimizer is independent of the model optimizer.

Example: 

```
optimizer = optim.Adam(model.parameters(), lr=0.01)
num_epochs = 2


### pytorch-metric-learning stuff ###
loss_func = losses.SubCenterArcFaceLoss(num_classes=10, embedding_size=128).to(device)
loss_optimizer = torch.optim.Adam(loss_func.parameters(), lr=1e-4)
```

I normally have used:

```
optimizer = optim.Adam(model.parameters()+(loss_func.parameters(),, lr=some_cyclic_lr)
```

Are they both equivalent? Is one better than the other?

Thank you for the awesome pkg :D

**Comment by KevinMusgrave** [comment]
I think they're equivalent, and that it's just a difference in code style and organization. For example, it's easier to do selective optimization (e.g. optimize just the loss function's parameters) with separate optimizers. On the other hand, having everything in a single optimizer can simplify code.

**Comment by abaqusstudent** [comment]
Apologies for the delayed response. Thank you for your reply and clarification :)
